README.md

```diff
@@ -188,9 +188,9 @@ expect(parentElement).toContainHTML('<span data-testid="child"></span>')
 ```
 
 > Chances are you probably do not need to use this matcher. We encourage testing from the perspective of how the user perceives the app in a browser. That's why testing against a specific DOM structure is not advised.
-
+> \
 > It could be useful in situations where the code being tested renders html that was obtained from an external source, and you want to validate that that html code was used as intended.
-
+> \
```

Comment:
I'm sorry for nit-picking, but why do you need the `\`? Without it you get separate praragraphs, which was my intention.